### PR TITLE
slashcommands added chad and request

### PR DESCRIPTION
**Pull request by Jonacodez**

### CODE REVIEW

File: main.lua [hunk 1]
```diff
@@ -61,6 +61,19 @@ client:on("slashCommandsReady", function()
...
 	end)
 	client:slashCommand(_history_cmd:finish())
+	
+	local _request_cmd = slash.new("request", "Request a link for a request.")
+	_request_cmd:callback(function(ia, params, cmd) ia:reply("https://github.com/Johnnynator/Jeff") 
+	end)
+
+	local _chad_cmd = slash.new("chad", "Determines if you're a chad.")
+	_chad_cmd:callback(function(ia, params, cmd)
+		if local name = ("Jona" OR "John") then
```
Comment (Johnnynator) [inline main.lua]:
```suggestion
		if args.name == "Jona" OR args.name == "John" then
```

**Review by Johnnynator — CHANGES_REQUESTED** [review]
You never register the commands e.g. with `client:slashCommand(_chad_cmd:finish())` nor do you register any parameters to `chad` (or `request`)

File: main.lua (hunk 1 above)
Comment (Jonacodez) [reply main.lua]: Changed according to owner

File: main.lua [hunk 2]
```diff
@@ -61,6 +61,21 @@ client:on("slashCommandsReady", function()
...
 	client:slashCommand(_history_cmd:finish())
+	
+	local _request_cmd = slash.new("request", "Request a link for a request.")
+	_request_cmd:callback(function(ia, params, cmd) ia:reply("https://github.com/Johnnynator/Jeff") 
+	end)
+	client:slashCommand(_request_cmd:finish())
+		
+	local _chad_cmd = slash.new("chad", "Determines if you're a chad.")
+	_chad_cmd:callback(function(ia, params, cmd)
+		if args.name == "Jona" OR args.name == "John" then
```
Comment (Johnnynator) [inline main.lua:73]: You still didn't define the option name, to get this working :)

**Pull request closed without merging**